static/src/js/components/MetadataForm/MetadataForm.jsx

```diff
@@ -138,8 +141,13 @@ const MetadataForm = () => {
 
   useEffect(() => {
     const { draft: fetchedDraft } = data || {}
+    const { revisionId: cmrRetrievedRevisionId } = fetchedDraft || ''
+
+    checkForCMRFetchDraftLag(cmrRetrievedRevisionId, appContextRevisionId)
+
     setOriginalDraft(fetchedDraft)
     setDraft(fetchedDraft)
+    setRevisionId()
```

Comment:
I would change this to setRevisionId(null) just to be more explicit that you are learning the old version id.